satpy/tests/reader_tests/test_fci_l2_nc.py

```diff
@@ -74,124 +96,124 @@ def setUp(self):
             two_layers_dataset[1, :, :] = 2 * np.ones((100, 10))
 
             mtg_geos_projection = nc.createVariable('mtg_geos_projection', int, dimensions=())
-            mtg_geos_projection.longitude_of_projection_origin = 10.0
+            mtg_geos_projection.longitude_of_projection_origin = 0.0
             mtg_geos_projection.semi_major_axis = 6378137.
-            mtg_geos_projection.semi_minor_axis = 6356752.
+            mtg_geos_projection.inverse_flattering = 298.257223563
             mtg_geos_projection.perspective_point_height = 35786400.
 
-        self.reader = FciL2NCFileHandler(
-            filename=self.test_file,
-            filename_info={
-                'creation_time':  datetime.datetime(year=2017, month=9, day=20,
-                                                    hour=12, minute=30, second=30),
-            },
-            filetype_info={}
-        )
+        self.fh = FciL2NCFileHandler(filename=self.test_file, filename_info={}, filetype_info={}
+                                     )
```

Comment:
```suggestion
        self.fh = FciL2NCFileHandler(filename=self.test_file, filename_info={}, filetype_info={})
```

Reply:
I have merged this and your other suggestions. It was the first time I merged someone's suggestions via github, but I think it worked as expected in the end.